test/integration_tflint_test.go

```diff
@@ -25,7 +27,9 @@ func TestTflintFindsNoIssuesWithValidCode(t *testing.T) {
 	assert.NotContains(t, errOut.String(), "Error while running tflint with args:")
 	assert.NotContains(t, errOut.String(), "Tflint found issues in the project. Check for the tflint logs above.")
 
-	assert.Contains(t, errOut.String(), fmt.Sprintf("--config %s/.tflint.hcl", modulePath))
+	found, err := regexp.MatchString(fmt.Sprintf("--config %s/.*/.tflint.hcl", modulePath), errOut.String())
```

Comment:
So this test wasn't testing?

Reply:
Before my changes, `.tflint.hcl` wasn't included in `.terragrunt-cache` directory and was loaded from "modulePath", with my changes it is copied to `.terragrunt-cache` but module path is hashed so I added regexp to match path